specification.json

```diff
@@ -11,6 +11,12 @@
             "cwd": "005-trim",
             "script": "./sbatch.sh"
         },
+        {
+            "name": "rrna",
```

Comment:
I think `rrna` and `mrna` should be added as dependencies to the `stop` step.  Otherwise there is a chance it will run before they are done. This could happen if all other steps didn't need to be run (because they've already been run in the past). In which case the `stop` step will not wait for `rrna` or `mrna` and will create the `slurm-pipeline.done` file before the pipeline has finished.

Reply:
Done